dev-tools/docker/ci/f24/Dockerfile

```diff
@@ -0,0 +1,30 @@
+FROM fedora:24
+MAINTAINER rpmdiff-maint@redhat.com
+
+### code ###
+VOLUME ["/code"]
```

Comment:
I pinched parts of this and did not realise that I don't need to specify it if I'm not sharing the containers around. Will remove.